WordPress/src/main/java/org/wordpress/android/ui/people/utils/PeopleUtils.java

```diff
@@ -115,6 +115,47 @@ public void onErrorResponse(VolleyError volleyError) {
         WordPress.getRestClientUtilsV1_1().get(path, params, null, listener, errorListener);
     }
 
+    public static void fetchViewers(final String blogId, final int localTableBlogId, final int offset,
+                                     final FetchViewersCallback callback) {
+        com.wordpress.rest.RestRequest.Listener listener = new RestRequest.Listener() {
+            @Override
+            public void onResponse(JSONObject jsonObject) {
+                if (jsonObject != null && callback != null) {
+                    try {
+                        JSONArray jsonArray = jsonObject.getJSONArray("viewers");
+                        List<Person> people = peopleListFromJSON(jsonArray, localTableBlogId, false, false,
+                                true);
+                        int numberOfUsers = jsonObject.optInt("found");
+                        boolean isEndOfList = (people.size() + offset) >= numberOfUsers;
+                        callback.onSuccess(people, isEndOfList);
+                    }
+                    catch (JSONException e) {
+                        AppLog.e(T.API, "JSON exception occurred while parsing the response for " +
+                                "sites/%s/viewers: " + e);
+                        callback.onError();
+                    }
+                }
+            }
+        };
+
+        RestRequest.ErrorListener errorListener = new RestRequest.ErrorListener() {
+            @Override
+            public void onErrorResponse(VolleyError volleyError) {
+                AppLog.e(T.API, volleyError);
+                if (callback != null) {
+                    callback.onError();
+                }
+            }
+        };
+
+        int page = (int) Math.floor(offset / FETCH_LIMIT) + 1;
```

Comment:
In some languages, it can round to the nearest integer instead of flooring it, so as a general practice I like to call `floor` explicitly to make it clear what I want to achieve in code. Changed it in: https://github.com/wordpress-mobile/WordPress-Android/pull/4383/commits/48e92141d122d0770f0f66bbee6e82c8bcaead44

P.S: This is another case of personal prefence, but oh well...